packages/buffetjs-styles/src/components/Table/index.js

```diff
@@ -78,6 +79,7 @@ const Table = styled.div`
       line-height: 0.1rem;
       font-weight: ${sizes.fontWeight.bold};
       text-transform: capitalize;
+      th,
       td {
```

Comment:
```suggestion
      th {
```

Question: Can `td` be removed here, or does this break custom tables in Strapi?

Reply:
You can remove it.